assets/scripts/segments/Segment.jsx

```diff
@@ -239,8 +239,8 @@ export class Segment extends React.Component {
     // Get localized names from store, fall back to segment default names if translated
     // text is not found. TODO: port to react-intl/formatMessage later.
     const displayName = segment.label || getLocaleSegmentName(segment.type, segment.variantString)
-    const capacity = getSegmentCapacity(segment).capacity.average
-    const showCapacity = enableAnalytics && capacity !== undefined
+    const { capacity: { average, display } } = getSegmentCapacity(segment)
+    const showCapacity = enableAnalytics && display
```

Comment:
This doesn't fix #1565 - let me know if you think this should be addressed in a separate PR?

Reply:
Just added a fix for this, since after some reflection, I think it's closely related.